troupon/payment/models.py

```diff
@@ -26,3 +27,18 @@ class TransactionHistory(models.Model):
                                        null=True,
                                        blank=True)
     user = models.ForeignKey(User)
+
+
+class IndividualTransactions(models.Model):
+    price = models.IntegerField()
+    advertiser = models.ForeignKey('deals.Advertiser')
+    title = models.CharField(max_length=100, null=False, blank=False)
+    description = models.TextField(blank=True, default='')
+    transaction_id = models.CharField(max_length=100,
```

Comment:
**Code Quality: Nitpick**

Let's change this to `stripe_transaction_id` just to be explicit that there are no relationships between this and any tables we created.